Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java

```diff
@@ -125,6 +125,22 @@ public void leaveSession(Player player, Session session) {
         controller.changeHost(name, room.getHost().getUsername());
         controller.decreaseOccupancy(name);
     }
+    
+    public void switchTeamFromPlayer(Player player) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)